lib/matplotlib/animation.py

```diff
@@ -1776,3 +1786,16 @@ def _draw_frame(self, framedata):
                 a.set_animated(self._blit)
 
     save_count = _api.deprecate_privatize_attribute("3.7")
+
+
+def _validate_grabframe_kwargs(savefig_kwargs):
+    if mpl.rcParams['savefig.bbox'] == 'tight':
+        raise ValueError(
+            "{mpl.rcParams['savefig.bbox']=} must not be 'tight' as it may cause frame "
+            "size to vary, which is inappropriate for animation."
```

Comment:
```suggestion
            f"{mpl.rcParams['savefig.bbox']=} must not be 'tight' as it may cause frame "
            f"size to vary, which is inappropriate for animation."
```